README.md

```
@@ -1,81 +1,131 @@
 # Official Flutter packages for [Stream Chat](https://getstream.io/chat/sdk/flutter/)
 
-![](https://raw.githubusercontent.com/GetStream/stream-chat-flutter/master/images/sdk_hero_v4.png)
+![](https://raw.githubusercontent.com/GetStream/stream-chat-flutter/master/images/sdk_hero_v10.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)